worlds/tunic/__init__.py

```diff
@@ -373,7 +385,8 @@ def fill_slot_data(self) -> Dict[str, Any]:
             "Hexagon Quest Holy Cross": self.ability_unlocks["Pages 42-43 (Holy Cross)"],
             "Hexagon Quest Icebolt": self.ability_unlocks["Pages 52-53 (Icebolt)"],
             "Hexagon Quest Goal": self.options.hexagon_goal.value,
-            "Entrance Rando": self.tunic_portal_pairs
+            "Entrance Rando": self.tunic_portal_pairs,
+            "disable_local_spoiler": int(tunic_settings["disable_local_spoiler"]),
```

Comment:
```suggestion
            "disable_local_spoiler": max(int(tunic_settings["disable_local_spoiler"]), int(self.multiworld.is_race)),
```
Since if the multiworld race mode is on, that's also a time to disable the internal spoiler log